Mathlib/Algebra/Order/CompleteField.lean

```diff
@@ -155,7 +155,8 @@ theorem cutMap_add (a b : α) : cutMap β (a + b) = cutMap β a + cutMap β b :=
       rw [coe_mem_cutMap_iff]
       exact_mod_cast sub_lt_comm.mp hq₁q
   · rintro _ ⟨_, _, ⟨qa, ha, rfl⟩, ⟨qb, hb, rfl⟩, rfl⟩
-    refine' ⟨qa + qb, _, by norm_cast⟩
+    -- After leanprover/lean4#2734, `norm_cast` needs help with beta reduction.
+    refine' ⟨qa + qb, _, by dsimp; norm_cast⟩
```

Comment:
Here and in all other places where you added a `dsimp`, should we use `beta_reduce` instead of `dsimp` to clearly say what this is doing?

Reply:
Done